week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -8,27 +8,26 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? (2 6 4 9 6 13 8 )
```

Comment:
2. 👍 
3. 👍 
4. 👍 
5. `f2` isn't called 5 times here - you can add a `console.log` into `f2` and everytime you see that `console.log` appear it will tell you how many times the function has run.
6. You can `console.log` `a` inside the function in order to work out the value of `a` when `f1` is first called.
7. You can do a similar thing here to work out this value.